### PR TITLE
Use new id cards module. FIST-785 #resolve

### PR DESCRIPTION
**Pull request by francisconeves97**

### CODE REVIEW

**Comment by francisconeves97** [comment]
This commit depends on https://github.com/ist-dsi/fenixedu-id-cards/pull/41

File: pom.xml [hunk 1]
```diff
@@ -40,7 +40,7 @@
         <version.org.fenixedu.messaging>4.0.1</version.org.fenixedu.messaging>
         <version.org.glassfish.jersey>2.27</version.org.glassfish.jersey>
         <version.pt.ist.drive-sdk>2.1.0</version.pt.ist.drive-sdk>
-        <version.pt.ist.fenixedu-id-cards>5.0.0</version.pt.ist.fenixedu-id-cards>
+        <version.pt.ist.fenixedu-id-cards>5.3.2</version.pt.ist.fenixedu-id-cards>
```
Comment (francisconeves97) [inline pom.xml]: This version should have https://github.com/ist-dsi/fenixedu-id-cards/pull/41

Comment (tiagoppinho) [reply pom.xml]: Will update to 5.4.0

**Comment by tiagoppinho** [comment]
The PR [ist-dsi/fenixedu-id-cards#41](https://github.com/ist-dsi/fenixedu-id-cards/pull/41) has been merged and the module has been released.

I updated this PR with the new id-cards version. This is ready to be merged.

**Pull request merged**